app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,37 @@
+ 
+<html>
+<body>
+    <div class="flex-centered bg-primary ctr-primary ">
+<div class=" flex-column rad-10 w-30">
+
+<%= form_for(resource, as: resource_name, url: user_session_path) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+  
+  <div class="form-field flex-column-centered mt-m10">
+
+    <div class="field flex-start">
+    <%= f.email_field :email, autofocus: true, autocomplete: "email", class: "input", placeholder: "Email"  %>
+    </div>
+
+    <div class="name-field flex-start">
+    <%= f.password_field :password, autocomplete: "current-password", class: "input", placeholder: "Password"  %>
+  </div>
+  
+  
+  <div class="actions flex-centered">
+  <%= f.submit "Login" %>
+  <% end %>
+  </div>
+  </div>
+  <div class="flex- mt-1 pl-1 flex-column-centered">
+  
+  <div class="flex">
+
+    <%= render "devise/shared/links" %>
+  </div>
```

Comment:
- [ ]  @kalbek great work on your login page, however, it requires a few adjustments to match the design given, below is a comparison and highlight of sections that need adjustings


Behance Design          |  Your design     
:-------------------------:|:-------------------------:
![image](https://github.com/kalbek/Budget-app/assets/61408860/ec0704f5-1a3f-48bc-9ce6-8777a321a1fb)  |  
![image](https://github.com/kalbek/Budget-app/assets/61408860/55c5331f-226b-40fc-bdd1-bc42be4a58ef)